cmd/clusterctl/client/cluster/mover.go

```diff
@@ -275,7 +275,7 @@ func getMoveSequence(graph *objectGraph) *moveSequence {
 		// NB. it is necessary to filter out nodes not belonging to a cluster because e.g. discovery reads all the secrets,
```

Comment:
In L120, do we need to check control plane ready condition instead of `clusterObj.Status.ControlPlaneReady `?
In my setup I don't see Status.controlPlaneReady anymore.

Reply:
It is required to install the CNI in order to get  Status.controlPlaneReady